### ISSUE TITLE
Postman will no longer load as of 6.1.2 (GPU issue)

### ISSUE THREAD

**Issue by JaggedJax** [a85/POSTMan-Chrome-Extension#4594]
This is on Windows 8.1 Pro. After updating to Postman 6.1.2 the standalone Windows app will no longer launch fully. I have tried to uninstall and re-install and the same issue persists.

The application window opens but the main content never loads. The File menu does start working after about 15 seconds, but selecting anything that affects the UI in the window or opens a new Postman window just leaves the window blank. Included are a series of screenshots showing how the UI is loading. I have been unable to find any log files so far, and the Postman Console will not load at all.

**Screenshots**
When first launched, the splash screen appears but it just blank:
![image](https://user-images.githubusercontent.com/553588/40457869-c4a5ef00-5ead-11e8-9bf0-1424546ecaa9.png)

Next the main window loads as plain white with nothing else displayed. This lasts for about 15-30 seconds:
![image](https://user-images.githubusercontent.com/553588/40457889-da2320e6-5ead-11e8-8688-158115e1b742.png)

Next the main window background beige color loads and the menu bar loads and works:
![image](https://user-images.githubusercontent.com/553588/40457909-ef7bc3f8-5ead-11e8-98bb-4912bdd16cbd.png)

The application will stay like this forever. Sometimes the UI will turn solid black. Also the Windows cursor pops up the loading icon about 4 times a second whenever hovering over the Postman UI. There are a bunch of Postman threads running, none of them are pegging the CPU, but they are all active and reach up to around 15% at times on my quad physical core PC.
![image](https://user-images.githubusercontent.com/553588/40458013-6b87ac96-5eae-11e8-9d75-6d677dc9abdc.png)

Here's what happens when I try opening the Postman Console:
![image](https://user-images.githubusercontent.com/553588/40458022-79c534c2-5eae-11e8-8c5f-085efd934eef.png)

**App information (please complete the following information):**
 - App Type: Native Windows App
 - Postman Version: 6.2.1
 - OS: Windows 8.1 x64

**Comment by harryi3t** [comment]
**UPDATE**: v.6.1.3 has been released which addresses this issue.  Download it from https://www.getpostman.com/apps and follow the steps below instead of using the canary build.

@JaggedJax We are tracking a known issue where Postman doesn’t launch with certain GPUs - we are investigating this at the moment and exploring possible fixes. In the meantime, please use the disable-gpu flag to continue using Postman. Please note that there is a possibility of performance degradation while this flag is set.

We have already added a workaround for this is the latest canary (https://www.getpostman.com/canary)
For it to work you will have to add an environment variable `POSTMAN_DISABLE_GPU=true`
We are in the process of adding documentation for this, for the time being you can follow these steps:

### 1. Open advanced system settings
![image](https://user-images.githubusercontent.com/5207331/40467461-aad4b7ec-5f47-11e8-9382-54383f4ca433.png)

### 2. In the `Advanced` tab click `Environment Variables`
![image](https://user-images.githubusercontent.com/5207331/40467512-db8cb114-5f47-11e8-9138-c043f7965771.png)

### 3. Add a new variable
![paul___postman_slack](https://user-images.githubusercontent.com/5207331/40467641-50ecb210-5f48-11e8-97ef-4a2728f36183.png)

### 4. Logout and Login back or Restart your PC

**Comment by JaggedJax** [comment]
This is working with that workaround. Additionally I saw that the latest on the website is now 6.0.10 and I can confirm that version works without the `--disable-gpu` flag as well.

If it's useful my GPU is an Intel HD Graphics 4000. The driver for it is super old (from October 21, 2013)

**Comment by harryi3t** [comment]
This workaround is available in latest version of Postman 6.1.3
@JaggedJax Thanks for the confirmation of the fix 👍

**Comment by JaggedJax** [comment]
I can confirm that the environmental variable is working in 6.1.3

**Comment by RandomDude777** [comment]
Can also confirm this fix is valid (win 8,1 sp1, postman 6.1.3)

**Comment by MG-CDS** [comment]
The work-around proposed by @harryi3t worked for me. I installed v6.1.3, added the suggested Environment Variable, and rebooted. Postman is now performing as expected.

Thank you @harryi3t ! :)

**Comment by harishrajora** [comment]
There is another going around that I did. You must be having NVIDIA or AMD processor. Maybe anything else. Go to the postman.exe location (which is in Users/<username>/App Data/local/postman/app/postman.exe

Right-click on it and run with intel HD graphics. It will run fine as before.

**Comment by aman-singh-parihar** [comment]
The work-around proposed by @harryi3t worked for me. 
Thanks @harryi3t

**Comment by atom0s** [comment]
Windows 8.1 Pro as well, desktop with a single GeForce GTX 970. This, to me, looks more like a Windows 8.1 issue than a problem with the graphics card. (I do not have a second card or integrated graphics for this to fall back on but the environment variable did work for me as well.)

**Comment by TheFrogPrince** [comment]
=o/  The environment variable works as a workaround.  The whole episode gets my vote for "Fail of the quarter", though.  You don't know much time I've wasted (including harassing McAfee administrators) before finding my way here.  Is it really necessary to try to squeeze out more performance by sitting directly on top of graphics cards?  (Windows 2012 R2 with dual AMD Radeon HD8490 cards).

**Comment by eugen-nw** [comment]
Neither the env. var, nor the "C:\Users\eugen\AppData\Local\Postman\app-6.1.3\Postman.exe --disable-gpu" command line parameter works for me.  I'm on Windows 10, using 6.1.3.  I see that I have an NVIDIA GeForce GTX 1070 video adapter.

The app starts up, displays its collections and after a short while its screen turns completely black.  Right click-ing Postman.exe in Windows Explorer does not give me the option to run with Intel HD graphics

**Comment by TurtleBeach** [comment]
Just installed 64-bit Postman 6.2.5 on Win7 64-bit Intel with Nvidia 510 Graphics Card / 4 monitors / Driver version 354.13; Postman would not run - same blank screen as referenced above, with the Windows "wait" cursor blinking rapidly (NOT rotating) after attempting to start Postman.
Adding the POSTMAN_DISABLE_GPU environment variable seems to have solved the problem.

**Comment by gmckeown** [comment]
Have just seen what I think is the same issue using v6.3.0 on Win2012R2 running under VMware. Disabling the GPU support worked; ran postman.exe with the --disable-gpu parameter.

**Comment by RyanGonet** [comment]
I also encountered this same issue with Postman v6.3.0 on Windows 8.1 Pro in an RDP session. Similarly worked around by passing the --disable-gpu argument.
Postman\Update.exe --processStart "Postman.exe" --process-start-args="--disable-gpu"

**Comment by erapkin** [comment]
I also am encountering this issue.  Windows 8.1 Enterprise, GPU Quadro K620 Driver Version 348.03, PostmanCanary 6.3.1 and Postman 6.3.0

**Comment by thomasmarton** [comment]
Also having this issue:
Windows 8.1 with a GTX 650 (driver version is 398.36)
(native, no RDP, VNC or anything similar involved, also I have the integrated GPU disabled so the nvidia is the only workhorse in my rig)
Tested both on 6.3.0 and canary 6.3.1 and both of them started working with the --disable-gpu argument.

**Comment by jawn** [comment]
Curious to know if this is issue could/should be fixed in the electron or postman code base?

**Comment by numaanashraf** [comment]
@jawn We are waiting for this to be fixed upstream in Chrome and then be pulled into Electron. We have an internal build ready with the latest version of Electron and are gathering feedback on the same. Would you be interested in helping us test this build?

**Comment by jawn** [comment]
Yes, reproduced on a colleague's computer, so can see if we can test it there.

**Comment by Staz00** [comment]
Experiencing this on the latest build (6.3.0), We are running Windows 8.1 Pro 32-bit with NVIDIA NVS 310 GPU.

But the system variable workaround still works for us

**Comment by ToniMaunde** [comment]
Thank you. This works on version 1.8.4.

**Comment by CanYouProgram** [comment]
If you go to your Postman.Exe and right click > Properties > Compatibility > Set compatibility to windows 8

This resolved the issue for me .

**Comment by ashansu-qainfotech** [comment]
> If you go to your Postman.Exe and right click > Properties > Compatibility > Set compatibility to windows 8
> 
> This resolved the issue for me .

@InnovationGB Thanks mate. This was a good one. Helped solve the problem :+1:

**Comment by drmcclelland** [comment]
> If you go to your Postman.Exe and right click > Properties > Compatibility > Set compatibility to windows 8
> 
> This resolved the issue for me .

This same workaround by @InnovationGB worked for me - I was having this same issue with Postman 6.4.4 installed on Windows Server 2012 R2 box that I have to connect to using Remote Desktop.

**Comment by Ramin-Bateni** [comment]
I had the same problem on **Win 8.1** with **Postman 6.4.4** and adding the **POSTMAN_GPU_DISABLE = false** solved my problem too.

**Comment by bsef15m002** [comment]
> If you go to your Postman.Exe and right click > Properties > Compatibility > Set compatibility to windows 8
> 
> This resolved the issue for me .

**Comment by MatGarreau** [comment]
workaround worked for me :-) Thank you @harryi3t

**Comment by shaunsational** [comment]
I just got the app for the first time (v6.4.4) and this issue was happening to me as well.  the system variable fixed the issue here too.  I'm running Win 8.1 on a Lenovo Y50-70 laptop if that helps.

Although I didn't have to logout/restart. I just added the variable and ran the app

**Comment by cbillins** [comment]
The environmental variable workaround worked for me. Currently using v6.5.2, Server 2012 R2.
@harryi3t Thanks!!

**Comment by brienRosenquist** [comment]
Workaround fixed the issue for me.
Windows Server 2012 R2
v6.5.2

**Comment by antoinecfmws** [comment]
Disabling GPU worked for me as well.  Solved using @RyanGonet.
Modify the shorcut on Windows and add the following at the end: 
`Postman\Update.exe --processStart "Postman.exe"` **--process-start-args="--disable-gpu**"

**Comment by dio-gh** [comment]
I've also encountered this exact issue, having updated after a really **really** long time. Setting an envvar fixed it for me as well, though I find it mind-boggling how such an issue could remain unresolved for ~six months. It's also a relatively severe one, as without the workaround the program won't properly open up for ~5 minutes in my experience.

@numaanashraf Are you guys still waiting for this to be fixed upstream or will there be a fix implemented? Is this issue even being investigated/worked on?

**Comment by shaunsational** [comment]
@nitrohigito @numaanashraf agreed, seems like the default should be to have the gpu disabled and have the envvar for enabling (not disabling) it, when so many are encountering the same issue with the same fix.

**Comment by janAtcontinia** [comment]
Adding the enviroment variable did not help me in version 6.5.3/windows 10.
Logging out of Postman and logging in again solved the problem.

**Comment by sandstones** [comment]
Still had to use the GPU workaround, December 2, 2018, version 6-5-3.  Being brand new to postman, having to do that to work around a bug that shows up as first reported back in 2016 makes me pretty dubious about the quality of this application.

**Comment by hebrayah** [comment]
I have brand new Postman and new laptop with clean operating system. The same issues/screens apply to my situation.

**Comment by larsts** [comment]
Each of the three proposed workarounds for disabling gpu worked for me on a 2012R2 server:
* Adding environment variable (did not require reboot)
* Setting Combatibility mode "Windows 8" on start menu shortcut
* Adding to shortcut start menu target: --process-start-args="--disable-gpu" 

While I'd contest this does not affect the quality of this application, I would suggest an opt-in solution for gpu might be warranted considering the number of users reporting this and similar issues.
I.e. changing the default not to enable gpu, but rather have an option of enabling gpu.

**Comment by JCMerrick** [comment]
I had the same problem with Postman on Windows 8.1 laptop.  It ran the 1st time, but started acting funky when I tried to re-launch the app the next morning.  As an absolute NOOB, all I can say is, "I.LOVE.YOU and THANK YOU! for taking your time to share with others."  I never take that for granted; in the (past) era of prima-donnas, folks would hold onto their fixes and insights to secure their employment-dominance.  THANK YOU!  I'm just a technical writer trying to update my skills to document APIs so I'm out of my element when programs don't cooperate as expected ;-)

**Comment by RevolvingDCON** [comment]
Hey guys could we get an official update on the progress of this bug? It's been open for half a year and I do believe it's quite critical as shortcut flags isn't really the best long term solution.

Thanks!

Edit: Please correct me if my opinion on the severity of this bug is incorrect.

**Comment by yegorandrosov** [comment]
I believe this Win issue, not Postman. I had everything working, but then I updated system using Win10 Update Assistant. Everything with GPU render broke - Steam, Slack, etc., so disable gpu flag should be attached to everything now. Interesting, that some games which had not worked before, started working after Win update.

maybe some testing can be done on launch. i guess you can get somehow info whether gpu render is available or not. and if it is not available, then reload with flag

**Comment by schivmeister** [comment]
> Disabling GPU worked for me as well. Solved using @RyanGonet.
> Modify the shorcut on Windows and add the following at the end:
> `Postman\Update.exe --processStart "Postman.exe"` **--process-start-args="--disable-gpu**"

I confirm that on Windows 8.1 Pro x64 this works. All the symptoms posted in the issue description are not present, and Postman loads quickly as usual.

Still to be seen whether everything works as expected, but at least I can start the new year without this annoyance, which had taken a toll on my workflow for the entirety of 2018!

**Comment by movAX13h** [comment]
Can someone who really knows the code fix it please? Fresh install on win8.1 and I get a blank window just like described here more than half a year ago.

**Comment by yegorandrosov** [comment]
@movAX13h no solution ever provided. you can try another windows image. reinstalling GPU drivers will never help (at least for me). I have to search for 'disable GPU render feature' in every application.
Mine PC broke after I updated Windows to latest version

**Comment by strikeflare** [comment]
Not sure if this is the same issue but it may have some similarities:
I downloaded Postman-win64-6.6.1-Setup.exe and installed it on a Windows Server 2012 R2 server. On startup, I got following error:
![image](https://user-images.githubusercontent.com/2511622/50834766-2ecf5b80-1355-11e9-83d4-c701669fa601.png)
After choosing Ok, the splashscreen won't disappear anymore and the GUI won't render either.

The workarounds described by @harryi3t, @larsts and others did not help for me. Neither after restarting the machine. 

Downgrading to Postman 6.0.10 works as expected.

**Comment by sorenhansen-dk** [comment]
> Not sure if this is the same issue but it may have some similarities:
> I downloaded Postman-win64-6.6.1-Setup.exe and installed it on a Windows Server 2012 R2 server. On startup, I got following error:
> ![image](https://user-images.githubusercontent.com/2511622/50834766-2ecf5b80-1355-11e9-83d4-c701669fa601.png)
> After choosing Ok, the splashscreen won't disappear anymore and the GUI won't render either.
> 
> The workarounds described by @harryi3t, @larsts and others did not help for me. Neither after restarting the machine.
> 

I'm having the exact same problem.

Running Windows 8.1 Pro x64

> Downgrading to Postman 6.0.10 works as expected.

Same here

**Update**

I can confirm that the canary release works with the environment variable `POSTMAN_DISABLE_GPU=true`

**Comment by nhhockeyplayer** [comment]
who is building this stuff

cannot even install it


?????????

**Comment by ZQ-PSK** [comment]
I'm on Windows 8 and running in Windows 8 compatibilty mode helped ;)

**Comment by bmercernccer** [comment]
This bug is still present in Postman 6.7.3. The environment variable hack works around the bug but it's definitely NOT fixed.

**Comment by itmajing** [comment]
After add an environment variable `POSTMAN_DISABLE_GPU=true` , the postman v6.7.3 works properly  on my PC which run a  windows 8.1  os

**Comment by alekseyl1992** [comment]
Had the same issue using Postman 6.7.4 on Windows 8.1.
`POSTMAN_DISABLE_GPU` helped

**Comment by m3dbedb** [comment]
@pwrigshi, how it could be a windows issue as long as the issue is a cross-platform and 'works' on windows since win7. Also peolpe say disabling gpu works on windows 8. On my windows 7 disabling gpu does not help.

**Comment by JeffWiebe** [comment]
Still getting blank screen on Postman startup. Tried uninstall, delete Local App Data, reinstall. Changed environment variable to DISABLE_GPU. Not in a Dev environment. No option to select which graphics processor.

OS Name	Microsoft Windows 10 Pro
Version	10.0.17763 Build 17763
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	xxxxx
System Manufacturer	LENOVO
System Model	42824YU
System Type	x64-based PC
System SKU	
Processor	Intel(R) Core(TM) i7-2860QM CPU @ 2.50GHz, 2501 Mhz, 4 Core(s), 8 Logical Processor(s)
BIOS Version/Date	LENOVO 8BET62WW (1.42 ), 2013-07-26
SMBIOS Version	2.6
Embedded Controller Version	1.36
BIOS Mode	Legacy
BaseBoard Manufacturer	LENOVO
BaseBoard Product	42824YU
BaseBoard Version	Not Available
Platform Role	Mobile
Secure Boot State	Unsupported
PCR7 Configuration	Binding Not Possible
Windows Directory	C:\WINDOWS
System Directory	C:\WINDOWS\system32
Boot Device	\Device\HarddiskVolume1
Locale	United States
Hardware Abstraction Layer	Version = "10.0.17763.194"
User Name	xxxxxx
Time Zone	Pacific Daylight Time
Installed Physical Memory (RAM)	16.0 GB
Total Physical Memory	15.9 GB
Available Physical Memory	7.95 GB
Total Virtual Memory	29.4 GB
Available Virtual Memory	20.1 GB
Page File Space	13.5 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	Off
Virtualization-based security	Not enabled
Device Encryption Support	Reasons for failed automatic device encryption: PCR7 binding is not supported, Hardware Security Test Interface failed and device is not InstantGo, Un-allowed DMA capable bus/device(s) detected, TPM is not usable
Hyper-V - VM Monitor Mode Extensions	Yes
Hyper-V - Second Level Address Translation Extensions	Yes
Hyper-V - Virtualization Enabled in Firmware	Yes
Hyper-V - Data Execution Protection	Yes

**Comment by lloy0076** [comment]
Honestly WHY does Postman even *need* a GPU - what's it doing? Crunching bitcoin in the background?

**Comment by sirvio** [comment]
black screen quite immediately after start of the program, "stack trace" attached when started the canary version from shell. Gpu parameters does not solve the problem.
"<--- Last few GCs --->

[6951:0x7f7ffb817e00]    22630 ms: Mark-sweep 2040.3 (2086.3) -> 2039.7 (2086.3) MB, 55.8 / 0.0 ms  allocation failure GC in old space requested
[6951:0x7f7ffb817e00]    22733 ms: Mark-sweep 2039.7 (2086.3) -> 2039.7 (2050.8) MB, 102.4 / 32.0 ms  last resort 
[6951:0x7f7ffb817e00]    22791 ms: Mark-sweep 2039.7 (2050.8) -> 2039.7 (2050.8) MB, 58.2 / 2.5 ms  last resort 


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 0x34b01e5f4661 <String[7]: file://>
    2: filter_record [file:///Applications/PostmanCanary.app/Contents/Resources/app/js/vendor-shared.js:173339] [bytecode=0x3bab6b1dab09 offset=40](this=0x1d7c7f008d79 <IDBRequest map = 0xa42a4426d9>)
    3: arguments adaptor frame: 1->0
    5: /* anonymous */ [file:///Applications/PostmanCanary.app/Contents/Resources/app/js/vendor-shared.js:169438] [bytecode=0x3bab6b1da7a1 offset=17](..."

this same thing was happening about one year ago, then changed to canary version, but now the problem is back.

============
==== EDIT ====
==> cleared all the cache stuff from "View -> Developer -> Dev Tools (current view) -> Clear Storage -> Clear site data"  See the screenshot. After this it is working again!

![Screenshot 2019-03-27 at 10 53 47](https://user-images.githubusercontent.com/21240165/55062429-a83a3880-507e-11e9-920b-be7130a32ad9.png)

**Comment by RobertTBoyerPhD** [comment]
I am on a Lenovo laptop, 64-bit, running Windows 8.1 and the solution above from "harryi3t commented on May 24, 2018" solved my problem.  Thank you.

**Comment by mihaj** [comment]
For the same reason (black screen after 1 min) I use canary version for more than 6 months now :). I think I tried GPU setting and it does not work. Still 7.0.7 version is not working, so I am always on the edge with Canaray :)

**Comment by kevinbosman** [comment]
Same issue with Postman 7.0.7:
Win8.1 Pro x64 (laptop)
NVIDIA GeForce GTX 860M + Intel HD 4600
Got it to work by setting Update.exe to Win 8 Compatibility mode

**Comment by SLE-TheRealOne** [comment]
Same problem with
* Postman 7.1.0
* Windows 7 Entreprise (6.1.7601)
* Core i5-6440HQ
* Intel HD Graphics 530

Solved the issue by using
`POSTMAN_DISABLE_GPU=true`

**Comment by harryi3t** [comment]
TL;DR: **We think we have a fix and need your help verifying this fix before we roll it out to everyone.**

Custom build link: https://drive.google.com/open?id=1TKEbwIH5ExK58bziaHSn6-_93GRjLYVS

We were able to trace the root cause of this to a combination of OS version + GPU hardware + Driver versions that did not work well with Electron/Chrome (which is internally used by Postman).
We are exploring an Electron version upgrade internally which we hope should mitigate these issues to a certain degree.

Since this issue is dependent on device configuration, we would need your help to verify this fix at your end before we roll the specific fix to all of our users.

We've created a custom build for this and to verify, please open the app and let us know if you are able to use it properly. Download link: https://drive.google.com/open?id=1TKEbwIH5ExK58bziaHSn6-_93GRjLYVS

Please let us know if this fixes things at your end and if it does not, it'll help a lot if you could send us your application logs (`View -> Developer -> View logs in Explorer`) to `help@getpostman.com` with the subject - "Logs for GitHub#4594"

Thanks for your help in getting this issue resolved.

PS: This build will ignore any workarounds you had done to your system to get Postman working (including the `POSTMAN_DISABLE_GPU` environment variable) and you needn't modify these workarounds to verify this build. Also, this built **will expire after a month**. Feel free to uninstall or delete this build after verifying.

Relevant issue on Electron project: https://github.com/electron/electron/issues/4380

**Comment by bmercernccer** [comment]
FYI, the download link requires permission to access.

**Comment by harryi3t** [comment]
@bmercernccer My bad, have fixed the permission. Can you try again?

**Comment by bmercernccer** [comment]
I have tried the test build briefly and didn't see any graphics issues with it. Fingers crossed!

**Comment by TurtleBeach** [comment]
Installation and cursory testing at localhost works. This on a Win7 Ultimate SP1 with NVidia NVS 510, Driver version 354.13, file version 6.14.13.5413 for all Display related code (NVidia control panel has different numbers).
"Normal" Postman requires disabling GPU on this machine (starting with Postman from a couple of years ago - 6.x but I do not remember precisely). So, if the Canary version truly ignores the GPU related environment variable, I'd say this one is OK.

**Comment by bmercernccer** [comment]
Have done some more testing of this release, so far no issues on Win7 or Win10 with Intel HD chipsets, and I did have issues before. 

I did change the POSTMAN_DISABLE_GPU environment variable to FALSE.

**Comment by juan-ramirez** [comment]
I can confirm that the env. variable workaround is still working as of today, issue is still present on 

Windows 8.1 - Intel Corei7 4710HQ CPU with NVidia GFORCE GTX 850M.

Thanks @harryi3t ,

Cheers!

**Comment by juan-ramirez** [comment]
@harryi3t Just tried the Particular build you attached a couple days ago and its working as expected 
Here are some logs if you wanna take a look

[PostmanCanary.zip](https://github.com/postmanlabs/postman-app-support/files/3258751/PostmanCanary.zip)

**Comment by eugenerr** [comment]
With or without POSTMAN_DISABLE_GPU variable set I couldn't start installers PostmanCanary-win64-7.1.1-canary07-Setup.exe or Postman-win64-7.1.1-Setup.exe using options Run -> Integrated Graphics or Run -> High Performance NVIDIA processor on Win 10 Lenovo T480 laptop.
Integrated Graphics is Intel UHD Graphics 620 and NVIDIA card is GeForce MX150. 

PostmanCanary-win64-7.1.1-custom-Setup.exe has installed application and I can confirm that application itself starts as designed.
When are you planning to make installer without 1 month limit available?

**Comment by ankurdengla** [comment]
@eugen-nw The fix for this issue is now available in the latest Canary app - [Postman Canary 7.3.0-canary01](https://www.getpostman.com/downloads/canary).
This build does not have the 1 month limit.

**Comment by eugenerr** [comment]
Hi @ankurdengla1996 thank you, downloaded v7.3.0-canary03, no problem installing!

**Comment by harryi3t** [comment]
@juan-ramirez @bmercernccer @TurtleBeach @eugenerr 
Thank you for verifying the fix on the custom build. We had released Postman 7.3.x with this fix to some of our users. During this partial rollout, we found a bug around TLS renegotiation. 
We have posted the detailed analysis here: https://github.com/postmanlabs/postman-app-support/issues/6886#issuecomment-510891488

We are looking into downgrading an internal dependency (Electron) to fix the TLS bug. We just wanted to make sure that because of this downgrade, the GPU issue should not get regressed.

**We need your help once again to verify this. 
The fix is available in on our canary channel (7.3.4-canary01). You can download it from** https://www.getpostman.com/downloads/canary

Thank you

**Comment by numaanashraf** [comment]
Thanks for your patience everyone. Postman v7.3.x includes an internal change that we believe should prevent this issue from happening to a large extent. I’ve included a detailed description below explaining why this happened and the further steps we are taking to minimize these issues going forward.

**Why was this happening?**
Postman’s Desktop Apps are built on top of Electron, an open-source desktop application framework which is also used by apps like Slack, VSCode, Skype, etc. Electron includes a packaged version of Chrome, which is instrumental in drawing the final GUI that we show inside a Postman window.

Chrome prefers using the on-device GPU as much as possible to perform accelerated rendering to draw the UI - but not all GPU hardware + driver version combinations work well with GPU acceleration. Chrome mitigates this issue to a certain degree by maintaining an empirically derived whitelist to turn off GPU acceleration for known combinations and these are kept updated in newer versions of Chrome.

Postman versions v6.1.x through 7.2.x used a version of Electron that internally used Chrome v59 (which did not include the latest version of this whitelist) and in Postman v7.3, we have updated the Electron version we use and we now ship with Chrome v66, which includes an updated version of the whitelist. This update took longer than we initially expected as we needed to make sure that this new version of Electron was stable enough for our entire user base.

**Next steps**
Due to the large number of GPU hardware + driver combinations out there, there might still be some devices where Postman might not render correctly. To support these devices, we are working on a way to detect these problems while Postman runs (through deeper integration with Electron & Chrome), inform our users and possibly disable GPU acceleration automatically.

In the interim, the recommended workaround in this thread will continue to work and we advise anyone still facing this issue to use this workaround (https://learning.getpostman.com/docs/postman/launching_postman/installation_and_updates/#troubleshooting-postman-updates) and help us stay on top of this issue by sharing your GPU & driver information here.

**References:**
GPU whitelist in Postman v6.1 - v7.2: https://chromium.googlesource.com/chromium/src/+/refs/tags/59.0.3071.115/gpu/config/software_rendering_list.json
GPU whitelist in Postman v7.3+ : https://chromium.googlesource.com/chromium/src/+/refs/tags/66.0.3359.181/gpu/config/software_rendering_list.json

**Comment by Redsandro** [comment]
[Chrome v66 means Electron 3](https://electronjs.org/blog/electron-3-0) which has a critical bug that on certain hardware [freezes the entire computer](https://github.com/postmanlabs/postman-app-support/issues/7393), making you lose all your work. The bug has been fixed in Electron 4 and up, but not Electron 3 because [Electron 3 is no longer supported](https://github.com/electron/electron/issues/19269#issuecomment-546921436).

**Comment by RyanHCCSI** [comment]
adding the POSTMAN_DISABLE_GPU=true environment variable resolved the Postman startup issue for me.

My equipment is a Lenovo P52s running Windows 10 w/ 2 display adapters: Intel UHD Graphics 620 and NVIDIA Quadro P500.

**Comment by Redsandro** [comment]
> POSTMAN_DISABLE_GPU=true

Where are these flags documented?

**Comment by haydenhancock** [comment]
Adding the POSTMAN_DISABLE_GPU=true did not work for me.

OS
- OS Microsoft Windows 10 Pro
- Version 10.0.18362 Build 18362
- Intel(R) Core(TM) i7-5820K CPU @3.30GHz
- 64 GB RAM

GPU
- GeForce GTX 980 Ti
- Driver version 441.66

Postman
- Version 7.14

**Comment by olistic** [comment]
> Due to the large number of GPU hardware + driver combinations out there, there might still be some devices where Postman might not render correctly. To support these devices, we are working on a way to detect these problems while Postman runs (through deeper integration with Electron & Chrome), inform our users and possibly disable GPU acceleration automatically.

Hey @numaanashraf! Did you end up finding an automated solution for this?

**Comment by eugen-nw** [comment]
It îs pitiful, I need to open dev. view right after I start Postman and hit F5 when the exception occurs if I do not want a black screen.

**Comment by zeeshanhaidergilani** [comment]
> Adding the POSTMAN_DISABLE_GPU=true did not work for me.
> 
> OS
> 
> * OS Microsoft Windows 10 Pro
> * Version 10.0.18362 Build 18362
> * Intel(R) Core(TM) i7-5820K CPU @3.30GHz
> * 64 GB RAM
> 
> GPU
> 
> * GeForce GTX 980 Ti
> * Driver version 441.66
> 
> Postman
> 
> * Version 7.14

Same here. Did not work for me either

**Comment by Dani21** [comment]
I ran into a similar issue, and none of the solutions in this thread worked. It also wouldn't close unless I killed the task, though choosing to enable/disable Harware Acceleration would restart it. I ended up using PowerShell instead.

Windows 10 Enterprise, Dell Latitude 5520
11th Gen Intel(R) Core(TM) i7-1185G7, Intel(R) Iris(R) Xe Graphics

![Postman with a white blank screen](https://user-images.githubusercontent.com/1804643/159048082-5cf1fcaf-b131-4486-8944-22b55d298147.png)

**Comment by numaanashraf** [comment]
@Dani21 Looks like you are impacted by https://github.com/postmanlabs/postman-app-support/issues/10763. Could you download the latest version of Postman from postman.com/downloads and see whether the issue is resolved for you?

**Comment by Dani21** [comment]
> @Dani21 Looks like you are impacted by #10763. Could you download the latest version of Postman from postman.com/downloads and see whether the issue is resolved for you?

Thanks, that was it!

**Comment by henryobiaraije** [comment]
I just updated to version 9.13.3 and started noticing this pixelation effect.

However, setting the environmental variable POSTMAN_DISABLE_GPU=true does solve the problem for me.

**Comment by ibahas** [comment]
> I ran into a similar issue, and none of the solutions in this thread worked. It also wouldn't close unless I killed the task, though choosing to enable/disable Harware Acceleration would restart it. I ended up using PowerShell instead.
> 
> Windows 10 Enterprise, Dell Latitude 5520 11th Gen Intel(R) Core(TM) i7-1185G7, Intel(R) Iris(R) Xe Graphics
> 
> ![Postman with a white blank screen](https://user-images.githubusercontent.com/1804643/159048082-5cf1fcaf-b131-4486-8944-22b55d298147.png)

Same Error 😥

**Comment by walidbosso** [comment]
> **UPDATE**: v.6.1.3 has been released which addresses this issue. Download it from https://www.getpostman.com/apps and follow the steps below instead of using the canary build.
> 
> @JaggedJax We are tracking a known issue where Postman doesn’t launch with certain GPUs - we are investigating this at the moment and exploring possible fixes. In the meantime, please use the disable-gpu flag to continue using Postman. Please note that there is a possibility of performance degradation while this flag is set.
> 
> We have already added a workaround for this is the latest canary (https://www.getpostman.com/canary) For it to work you will have to add an environment variable `POSTMAN_DISABLE_GPU=true` We are in the process of adding documentation for this, for the time being you can follow these steps:
> 
> ### 1. Open advanced system settings
> ![image](https://user-images.githubusercontent.com/5207331/40467461-aad4b7ec-5f47-11e8-9382-54383f4ca433.png)
> 
> ### 2. In the `Advanced` tab click `Environment Variables`
> ![image](https://user-images.githubusercontent.com/5207331/40467512-db8cb114-5f47-11e8-9138-c043f7965771.png)
> 
> ### 3. Add a new variable
> ![paul___postman_slack](https://user-images.githubusercontent.com/5207331/40467641-50ecb210-5f48-11e8-97ef-4a2728f36183.png)
> 
> ### 4. Logout and Login back or Restart your PC

Well done!

**Comment by walidbosso** [comment]
Using Windows8.1, This works like charm, thanks @harryi3t
